coalib/parsing/CliParsing.py

```diff
@@ -58,7 +58,7 @@ def parse_cli(arg_list=None,
                                   line_parser)
         else:
             if isinstance(arg_value, list):
-                arg_value = ','.join([str(val) for val in arg_value])
+                arg_value = ','.join(str(val) for val in arg_value)
```

Comment:
could you make single commits for each file you modify? as the kind of refactor does differ from file to file